README.md

```diff
@@ -1,3 +1,37 @@
 # go-library is a monorepo of modules
 
 The go-library is a monorepo of golang modules designed for reusability within CLI and daemon applications.
+
+## Conventions
+
+A module should be self contained within its own folder off the root of the repository.
```

Comment:
Make sure to distinguish between a "package" and a "module" (contains a go.mod).
```suggestion
A package should be self contained within its own folder off the root of the repository.
```